### PR TITLE
[FrameworkBundle] Move lint commands to lint namespace.

### PR DESCRIPTION
**Pull request by aitboudad**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Fixed tickets | ~ |
| Tests pass? | yes |
| License | MIT |

### CODE REVIEW

File: src/Symfony/Bridge/Twig/Command/LintCommand.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ class LintCommand extends Command
     /**
      * {@inheritdoc}
      */
-    public function __construct($name = 'twig:lint')
+    public function __construct($name = 'lint:twig')
```
Comment (xabbuh) [inline src/Symfony/Bridge/Twig/Command/LintCommand.php]: Without an alias, this will break now for people relying on the old name.

Comment (aitboudad) [inline src/Symfony/Bridge/Twig/Command/LintCommand.php]: fixed

**Comment by stof** [comment]
This makes sense to me, just like we grouped the debug commands

**Comment by fabpot** [comment]
@aitboudad You should also trigger a deprecation notice when the old name is used (we've done that already for the debug:* commands).

**Comment by aitboudad** [comment]
@fabpot done

**Comment by fabpot** [comment]
Thank you @aitboudad.

**Comment by fabpot** [comment]
@aitboudad This one broke the tests :( Can you have a look at that ASAP? Thanks.

**Comment by aitboudad** [comment]
ok

**Comment by aitboudad** [comment]
@fabpot fixed by #14197

**Pull request merged**